conduitqe/tests/api/test_facts.py

```diff
@@ -0,0 +1,97 @@
+"""Tests for triggering facts updates.
+
+:caseautomation: automated
+:casecomponent: api
+:caseimportance: high
+:caselevel: integration
+:requirement: rhsm-conduit
+:testtype: functional
+:upstream: yes
+"""
+
+import logging
+import json
+import time
+import pytest
+import oc
+
+from conduitqe.config import get_config
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def rhsm_conduit_instance():
+    cfg = get_config()
+    return cfg.pod
+
+
+@pytest.fixture(scope="module")
+def config():
+    cfg = get_config()
+    return cfg
+
+
+@pytest.fixture(scope="module")
+def openshift_setup():
```

Comment:
You could use the `config` fixture here so you don't have to call `get_config` inside this function again.